docs/en/observability/apm/configure/env.asciidoc

```diff
@@ -95,4 +95,27 @@ output.elasticsearch:
 When APM Server loads the config file, it resolves the environment variable and
 replaces it with the specified list before reading the `hosts` setting.
 
-NOTE: Do not use double-quotes (`"`) to wrap regular expressions, or the backslash (`\`) will be interpreted as an escape character.
\ No newline at end of file
+[float]
+=== Opt-out of complex object parsing
```

Comment:
[to reviewer] feel free to suggest a better header name, or whether a header makes sense here

Reply:
this seems good 👍